### ISSUE TITLE
parasoll mqtt no reading available open/close

### ISSUE THREAD

**Issue by hyper2910** [dresden-elektronik/deconz-rest-plugin#7507]
### Does the issue really belong here?

- [X] I definitively want to report a bug within deCONZ or its REST-API

### Is there already an existing issue for this?

- [X] I have searched the existing issues and there is none for the bug at hand

### Describe the bug

no status change via mqtt

STATE is evrytime INITIALIZED

as Reading also only the following coming

![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/35178004/6a7a3423-5c28-4e3a-b236-db86e000fc59)

### Steps to reproduce the behavior

deleting and new reconnection of device:

### Expected behavior

a status/reading with open/close

### Environment

- Host system: Laptop
- Running method: Mate 1,24,1
- Firmware version: 26720700  updated to 26780700
- deCONZ version (not Home assistant Addon version!): 2.25.1
- Device:      ConBee II
- Do you use an USB extension cable: (yes / no) -- only relevant for ConBee I/II/III   YES
- Is there any other USB or serial devices connected to the host system? If so: Which?   Yes,Homematic USB , 5 USB CUL for different Systems

### deCONZ Logs

REST API
```
{
    "config": {
        "battery": 94,
        "on": true,
        "pending": [],
        "reachable": true
    },
    "ep": 2,
    "etag": "d0bbefea005e36de397817e977ea2140",
    "lastannounced": null,
    "lastseen": "2024-01-02T11:13Z",
    "manufacturername": "IKEA of Sweden",
    "modelid": "PARASOLL Door/Window Sensor",
    "name": "PARASOLL Door/Window Sensor",
    "state": {
        "lastupdated": "none",
        "lowbattery": null,
        "open": null
    },
    "swversion": "1.0.19",
    "type": "ZHAOpenClose",
    "uniqueid": "04:87:27:ff:fe:b8:a7:1c-02-0500"
}.
```

Events after new connecting and some open and closes:
```
{
    "12:13:36:663": {
        "attr": {
            "id": "15",
            "lastannounced": null,
            "lastseen": "2024-01-02T11:13Z",
            "manufacturername": "IKEA of Sweden",
            "mode": 1,
            "modelid": "PARASOLL Door/Window Sensor",
            "name": "PARASOLL Door/Window Sensor",
            "swversion": "1.0.19",
            "type": "ZHAOpenClose",
            "uniqueid": "04:87:27:ff:fe:b8:a7:1c-02-0500"
        },
        "e": "changed",
        "id": "15",
        "r": "sensors",
        "t": "event",
        "uniqueid": "04:87:27:ff:fe:b8:a7:1c-02-0500"
    },
    "12:14:36:863": {
        "attr": {
            "id": "15",
            "lastannounced": null,
            "lastseen": "2024-01-02T11:14Z",
            "manufacturername": "IKEA of Sweden",
            "mode": 1,
            "modelid": "PARASOLL Door/Window Sensor",
            "name": "PARASOLL Door/Window Sensor",
            "swversion": "1.0.19",
            "type": "ZHAOpenClose",
            "uniqueid": "04:87:27:ff:fe:b8:a7:1c-02-0500"
        },
        "e": "changed",
        "id": "15",
        "r": "sensors",
        "t": "event",
        "uniqueid": "04:87:27:ff:fe:b8:a7:1c-02-0500"
    }
}

```

DDF Rest API

```
{
    "schema": "devcap1.schema.json",
    "manufacturername": "$MF_IKEA",
    "modelid": "PARASOLL Door/Window Sensor",
    "product": "PARASOLL Door/Window Sensor",
    "sleeper": true,
    "status": "Gold",
    "path": "/devices/ikea/parasoll_open_close_sensor.json",
    "subdevices": [
        {
            "type": "$TYPE_OPEN_CLOSE_SENSOR",
            "restapi": "/sensors",
            "uuid": [
                "$address.ext",
                "0x02",
                "0x0500"
            ],
            "fingerprint": {
                "profile": "0x0104",
                "device": "0x0402",
                "endpoint": "0x02",
                "in": [
                    "0x0000",
                    "0x0001",
                    "0x0500"
                ]
            },
            "items": [
                {
                    "name": "attr/id"
                },
                {
                    "name": "attr/lastannounced"
                },
                {
                    "name": "attr/lastseen"
                },
                {
                    "name": "attr/manufacturername"
                },
                {
                    "name": "attr/modelid"
                },
                {
                    "name": "attr/name"
                },
                {
                    "name": "attr/swversion"
                },
                {
                    "name": "attr/type"
                },
                {
                    "name": "attr/uniqueid"
                },
                {
                    "name": "config/battery",
                    "awake": true,
                    "read": {
                        "fn": "none"
                    },
                    "parse": {
                        "at": "0x0021",
                        "cl": "0x0001",
                        "ep": 1,
                        "eval": "Item.val = Attr.val / 2;",
                        "fn": "zcl:attr"
                    },
                    "default": 0
                },
                {
                    "name": "config/checkin",
                    "public": false,
                    "refresh.interval": 3600,
                    "read": {
                        "at": "0x0000",
                        "cl": "0x0020",
                        "ep": 1,
                        "fn": "zcl:attr"
                    },
                    "write": {
                        "at": "0x0000",
                        "cl": "0x0020",
                        "dt": "0x23",
                        "ep": 1,
                        "eval": "Item.val",
                        "fn": "zcl:attr"
                    },
                    "parse": {
                        "at": "0x0000",
                        "cl": "0x0020",
                        "ep": 1,
                        "eval": "Item.val = Attr.val",
                        "fn": "zcl:attr"
                    },
                    "default": 14400
                },
                {
                    "name": "config/enrolled",
                    "public": false
                },
                {
                    "name": "config/on"
                },
                {
                    "name": "config/pending"
                },
                {
                    "name": "config/reachable"
                },
                {
                    "name": "state/lastupdated"
                },
                {
                    "name": "state/lowbattery",
                    "awake": true
                },
                {
                    "name": "state/open",
                    "awake": true
                }
            ]
        }
    ],
    "bindings": [
        {
            "bind": "unicast",
            "src.ep": 1,
            "cl": "0x0001",
            "report": [
                {
                    "at": "0x0021",
                    "dt": "0x20",
                    "min": 3600,
                    "max": 43200,
                    "change": "0x00000002"
                }
            ]
        },
        {
            "bind": "unicast",
            "src.ep": 2,
            "cl": "0x0500"
        }
    ]
}

```
### Additional context

.

**Comment by hyper2910** [comment]
also no "wert" in P
![Screenshot 2024-01-02 072310](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/35178004/441b4445-1fff-4b5a-bf7f-6314067a2d74)
hoscon

**Comment by Mimiix** [comment]
Hi,

what does the RestAPI / Websocket show?

**Comment by hyper2910** [comment]
Hi, 
what do you mean?

in phoscon also no "Status/WERT" is available.


![parasoll](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/35178004/ad49fb19-9250-45c1-a7d8-18839e60f1bb)

**Comment by Mimiix** [comment]
Hi,

You can check in Phoscon under Help > API information. There, filter for Sensors and select your Parasoll sensor.

If you open/close it, does it show changes under Events tab? Can you also show the Rest API output?

**Comment by hyper2910** [comment]
Sensor Open:
```
{
    "config": {
        "battery": 94,
        "on": true,
        "pending": [],
        "reachable": true
    },
    "ep": 2,
    "etag": "cfc8a9095086bb68996bf611ce4ebb33",
    "lastannounced": null,
    "lastseen": "2024-01-02T08:46Z",
    "manufacturername": "IKEA of Sweden",
    "modelid": "PARASOLL Door/Window Sensor",
    "name": "OpenClose_test",
    "state": {
        "lastupdated": "none",
        "lowbattery": null,
        "open": null
    },
    "swversion": "1.0.19",
    "type": "ZHAOpenClose",
    "uniqueid": "04:87:27:ff:fe:b8:a7:1c-02-0500"
}
```
Sensor Closed:
`{
    "config": {
        "battery": 94,
        "on": true,
        "pending": [],
        "reachable": true
    },
    "ep": 2,
    "etag": "cfc8a9095086bb68996bf611ce4ebb33",
    "lastannounced": null,
    "lastseen": "2024-01-02T08:46Z",
    "manufacturername": "IKEA of Sweden",
    "modelid": "PARASOLL Door/Window Sensor",
    "name": "OpenClose_test",
    "state": {
        "lastupdated": "none",
        "lowbattery": null,
        "open": null
    },
    "swversion": "1.0.19",
    "type": "ZHAOpenClose",
    "uniqueid": "04:87:27:ff:fe:b8:a7:1c-02-0500"
}`

Unter Events:
{}
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/35178004/eb6abb7c-4b50-455e-be9c-f96c893aa2e1)

**Comment by Mimiix** [comment]
Can you please fill the details of the template correctly? 

Also, can you show me a screenshot of the basic cluster of the device in deconz?

https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/How-to-read-Clusters

**Comment by hyper2910** [comment]
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/35178004/4d4604c9-845e-4c8a-ab66-6b49f33067a3)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/35178004/f9777d0a-dbba-414f-8706-3a8d2f9ca419)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/35178004/f6e6f799-09e4-47e4-8636-90eb7f877b12)

**Comment by Mimiix** [comment]
Still waiting on the template info.

**Comment by hyper2910** [comment]
sorry my first time in github.

What do you need more?

**Comment by Mimiix** [comment]
The template asks for environment details. You need to fill those.

**Comment by hyper2910** [comment]
where did i get the template back?

**Comment by Mimiix** [comment]
> where did i get the template back?

Edit your first post.

**Comment by hyper2910** [comment]
> > where did i get the template back?
> 
> Edit your first post.

Done

**Comment by Mimiix** [comment]
didyou try to re-pair the sensor already?

**Comment by hyper2910** [comment]
> didyou try to re-pair the sensor already?

Yes,  done a few minutes ago and same issues

**Comment by Mimiix** [comment]
https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually

You might need to update the firmware on the conbee as it is outdated.

**Comment by Mimiix** [comment]
Can you:

Remove the device,
Restart deconz
Start logging with log levels APS, APS L2, ERROR, ERROR L2, INFO, INFO L2. You can log in deconz under help , debug logs.
Start searching for new devices.
After pairing share the logs via Pastebin

**Comment by hyper2910** [comment]
aftre repairing the "Status" is available.
many thanks.

**Comment by Mimiix** [comment]
Happy to hear it works. Take care :)

**Comment by tkoenig** [comment]
I'm having the same issues with this device. 
I tried the same steps with removing the device and restarting and pairing it again but no luck.

@hyper2910 does it now show anything under "Werte" for you now?
@Mimiix I tried pairing this device before upgrading to the beta release, could this cause an issue?
@ebaauw I noticed from another issue that you also seem to have this device, does it work for you? (I'm using `homebridge-deconz`, where the device shows up, but also no state changes).

Below is what I get in the AI Information (with a little bit of a lag):

<details>
<summary>Events</summary>

```
{
    "20:25:16:161": {
        "attr": {
            "id": "28",
            "lastannounced": "2024-01-18T19:24:16Z",
            "lastseen": "2024-01-18T19:25Z",
            "manufacturername": "IKEA of Sweden",
            "mode": 1,
            "modelid": "PARASOLL Door/Window Sensor",
            "name": "PARASOLL Door/Window Sensor",
            "swversion": "1.0.19",
            "type": "ZHAOpenClose",
            "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
        },
        "e": "changed",
        "id": "28",
        "r": "sensors",
        "t": "event",
        "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
    },
    "20:26:16:476": {
        "attr": {
            "id": "28",
            "lastannounced": "2024-01-18T19:24:16Z",
            "lastseen": "2024-01-18T19:26Z",
            "manufacturername": "IKEA of Sweden",
            "mode": 1,
            "modelid": "PARASOLL Door/Window Sensor",
            "name": "PARASOLL Door/Window Sensor",
            "swversion": "1.0.19",
            "type": "ZHAOpenClose",
            "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
        },
        "e": "changed",
        "id": "28",
        "r": "sensors",
        "t": "event",
        "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
    },
    "20:27:33:893": {
        "attr": {
            "id": "28",
            "lastannounced": "2024-01-18T19:24:16Z",
            "lastseen": "2024-01-18T19:27Z",
            "manufacturername": "IKEA of Sweden",
            "mode": 1,
            "modelid": "PARASOLL Door/Window Sensor",
            "name": "PARASOLL Door/Window Sensor",
            "swversion": "1.0.19",
            "type": "ZHAOpenClose",
            "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
        },
        "e": "changed",
        "id": "28",
        "r": "sensors",
        "t": "event",
        "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
    },
    "20:28:34:208": {
        "attr": {
            "id": "28",
            "lastannounced": "2024-01-18T19:24:16Z",
            "lastseen": "2024-01-18T19:28Z",
            "manufacturername": "IKEA of Sweden",
            "mode": 1,
            "modelid": "PARASOLL Door/Window Sensor",
            "name": "PARASOLL Door/Window Sensor",
            "swversion": "1.0.19",
            "type": "ZHAOpenClose",
            "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
        },
        "e": "changed",
        "id": "28",
        "r": "sensors",
        "t": "event",
        "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
    },
    "20:34:18:966": {
        "attr": {
            "id": "28",
            "lastannounced": "2024-01-18T19:24:16Z",
            "lastseen": "2024-01-18T19:34Z",
            "manufacturername": "IKEA of Sweden",
            "mode": 1,
            "modelid": "PARASOLL Door/Window Sensor",
            "name": "PARASOLL Door/Window Sensor",
            "swversion": "1.0.19",
            "type": "ZHAOpenClose",
            "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
        },
        "e": "changed",
        "id": "28",
        "r": "sensors",
        "t": "event",
        "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
    }
}
```
</details>
<details>
<summary>RestAPI</summary>

```
{
    "config": {
        "battery": 0,
        "on": true,
        "pending": [],
        "reachable": true
    },
    "ep": 2,
    "etag": "470548709c94e4c049515398cfd53fdd",
    "lastannounced": "2024-01-18T19:24:16Z",
    "lastseen": "2024-01-18T19:34Z",
    "manufacturername": "IKEA of Sweden",
    "modelid": "PARASOLL Door/Window Sensor",
    "name": "PARASOLL Door/Window Sensor",
    "state": {
        "lastupdated": "none",
        "lowbattery": null,
        "open": null
    },
    "swversion": "1.0.19",
    "type": "ZHAOpenClose",
    "uniqueid": "3c:2e:f5:ff:fe:42:4c:f9-02-0500"
}
```
</details
